### DISCUSSION TITLE
Coverage Map

### DISCUSSION THREAD

**Discussion by shiv20032** [NVlabs/sionna#656] [Q&A] [answered]
![Screenshot from 2024-11-11 16-11-33](https://github.com/user-attachments/assets/f568083e-0eea-49f2-8e07-cab5cd95ff9d)
![image](https://github.com/user-attachments/assets/c44028fb-936f-44bb-b253-af320603d5dc)

```py
tx0 = Transmitter(name='tx0',
                  position=[0, 0, 20],
                  orientation=[0, 0, 0],
                  power_dbm=44)
```

why orientation effect this view? suppose I want to see filed without orientation change of TX, how to do the same?

**Comment by faycalaa** [comment, marked as answer]
Hi @shiv20032,

As you are using a directive antenna pattern (tr38901), the orientation of the transmitter impacts the coverage map.

**Reply by shiv20032** [reply]
Thanks for the reply.

My question is why np.pi? can i just change camera angle and see the pattern of tr38901?

**Reply by faycalaa** [reply]
No camera is used to visualize the coverage map in the code you are sharing. Note that you can visualize the antenna pattern using the [visualize utility of Antenna](https://nvlabs.github.io/sionna/api/rt.html#sionna.rt.antenna.visualize). You can also visualize the coverage map laid out in the scene using [Scene.preview](https://nvlabs.github.io/sionna/api/rt.html#sionna.rt.Scene.preview).
